pkg/kratos/handlers.go

```diff
@@ -279,6 +326,27 @@ func (a *API) mfaSettingsRedirect(w http.ResponseWriter) {
 	)
 }
 
+func (a *API) lookupSecretsSettingsRedirect(w http.ResponseWriter, flowId string) {
+	redirectUrl, err := url.Parse(path.Join("/", a.contextPath, "/ui/backup_codes_regenerate?flow="+flowId))
```

Comment:
nit-pick: use the `UI` constant found in the `ui` package instead of hardcoding the `ui` path?